staging/src/k8s.io/kubectl/pkg/cmd/apply/patcher.go

```diff
@@ -106,13 +104,13 @@ func (p *Patcher) patchSimple(obj runtime.Object, modified []byte, namespace, na
 	// Serialize the current configuration of the object from the server.
 	current, err := runtime.Encode(unstructured.UnstructuredJSONScheme, obj)
 	if err != nil {
-		return nil, nil, errors.Wrapf(err, "serializing current configuration from:\n%v\nfor:", obj)
+		return nil, nil, fmt.Errorf("serializing current configuration from:\n%v\nfor:, err: %w", obj, err)
```

Comment:
I think the resulting message is still different, i.e.:

```sh
# before
serializing current configuration from:
{obj}
for:: the original error

# after
serializing current configuration from:
{obj}
for:, err: the original error
```

The double colon in the original isn't desirable, but I think we want `for: %w` rather than  `for:, err: %w`